### ISSUE TITLE
Breaking 3d pie chart

### ISSUE THREAD

**Issue by sunilvarmach** [highslide-software/highcharts.com#3323]
slice is breaking in pie chart
![3d pie chart](https://cloud.githubusercontent.com/assets/8371706/3837747/9b2d0484-1df6-11e4-8373-5496c1badcf1.JPG)
Code:
```javascript
$(function () {
    $('#container').highcharts({
        chart: {
            type: 'pie',
          reflow: false,
            options3d: {
				enabled: true,
                alpha: 85,
                beta: 0
            }
        },
      legend: {
                enabled: true,
                align: 'bottom',
                layout: 'vertical',
                verticalAlign: 'bottom'
            },
        title: {
            text: 'Browser market shares at a specific website, 2014'
        },
        tooltip: {
            pointFormat: '{series.name}: <b>{point.percentage:.1f}%</b>'
        },
        plotOptions: {
            pie: {
              size:400,
                allowPointSelect: true,
                cursor: 'pointer',
                depth: 130,
                animation: false,
                    //minSize: 1,
                    dataLabels: {
                        enabled: false,
                        format: '{point.y}: {point.percentage:.1f} % asd'
                        //softConnector: false,
                        //connectorPadding: 5,
                        //distance: 30
                    },
                    showInLegend: true
            }
        },
        series: [{
            type: 'pie',
            data: [
                    { "name": "FFI PREMIER INSTITUTIONAL FUND", "y": 5600.11, "sliced": true, "selected": true },
                    { "name": "2001 ML VENTURES", "y": 8612.1, "sliced": true, "selected": true}]
        }]
    });
});
```

**Comment by sebastianbochan** [comment]
+1

**Comment by highslide-software** [comment]
@stephanevanraes Sebastian's example from November is reproduceable in Chrome.

**Comment by highslide-software** [comment]
Closed by 7a6232771746

**Comment by highslide-software** [comment]
@KacperMadej Thanks for the fix! Note that the issue wastn't closed automatically because you put a semicolon between "Fixed" and the issue number.
